src/manage_pg.c

```diff
@@ -3218,6 +3225,9 @@ create_tables ()
   sql ("SELECT create_index ('nvts_by_solution_type',"
        "                     'nvts', 'solution_type');");
 
+  sql ("SELECT create_index ('vt_refs_by_vt_oid',"
```

Comment:
This would be better after tag, because they're alphabetical (except for result and report, for some reason).

Reply:
Fixed.